_includes/pbc/all/install-features/202410.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -250,6 +250,8 @@ Set up the following behaviors.
 | MerchantUserUserRoleFilterPlugin                                 | Filters `ROLE_BACK_OFFICE_USER` to prevent a merchant user from logging into the Back Office.                           |               | Spryker\Zed\MerchantUser\Communication\Plugin\SecurityGui                        |
 | ProductViewerForOfferCreationAclInstallerPlugin                  | Provides `ProductViewerForOfferCreation` roles with rules and groups to create on installation.                         |               | Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser                  |
 | AclGroupMerchantUserLoginRestrictionPlugin                       | Checks if the merchant user login is restricted.                                                                        |               | Spryker\Zed\AclMerchantPortal\Communication\Plugin\SecurityMerchantPortalGui     |
+| MailMerchantUserPasswordResetRequestStrategyPlugin | Checks if strategy is applicable for a password reset request. Returns true if merchant user exists, false otherwise. |  | Spryker\Zed\MerchantUserPasswordResetMail\Communication\Plugin\UserPasswordReset  |
+| MerchantUserPasswordResetMailTypePlugin | Builds the mail of merchant password restore type. |  | Spryker\Zed\MerchantUserPasswordResetMail\Communication\Plugin\Mail  |
```

Comment:
Does it create an email?

Reply:
It creates a transfer that is further used to create email. Rephrased a bit.